assets/css/sensei-theme-blocks.scss

```diff
@@ -7,5 +7,6 @@
 @import 'sensei-course-theme/blocks/course-title';
 @import 'sensei-course-theme/blocks/exit-course-button';
 @import 'sensei-course-theme/blocks/contact-teacher';
+@import 'sensei-course-theme/blocks/exit-course';
```

Comment:
We already have another file to customize it, https://github.com/Automattic/sensei/blob/f12e3f9dde65609be1ee1f51039b14fbc6692a78/assets/css/sensei-course-theme/blocks/exit-course-button.scss

That is included  [here](https://github.com/Automattic/sensei/blob/f12e3f9dde65609be1ee1f51039b14fbc6692a78/assets/css/sensei-theme-blocks.scss#L8) 

Should we merge the selectors?

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6783/commits/da6e867ce3f25e5d0482d599ee3ffc3f15f5af6c/

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6783/commits/a829ff887961a4dc85e634966fe827a91c92bd94